docs/docs/Clients.md

```diff
@@ -3,7 +3,7 @@ title: "Client Libraries"
 linkTitle: "Clients"
```

Comment:
We should remove the clients.

Reply:
Hi Pieter. Apologies, but I'm not clear what you're asking me to do with this comment. Would you please clarify for me?

Reply:
We should not have client library specific pages for modules on redis.io. We can keep this part of the readme of the module repository

Reply:
@dwdougherty don't worry about this one, it's a part of my fork

Reply:
Thanks Elena. Resolving.